src/clj/puppetlabs/services/jruby/jruby_puppet_agents.clj

```diff
@@ -1,34 +1,21 @@
 (ns puppetlabs.services.jruby.jruby-puppet-agents
-  (:import (clojure.lang IFn Agent)
-           (puppetlabs.services.jruby.jruby_puppet_core PoisonPill RetryPoisonPill)
-           (com.puppetlabs.puppetserver PuppetProfiler))
+  (:import (clojure.lang IFn)
+           (com.puppetlabs.puppetserver PuppetProfiler)
+           (puppetlabs.services.jruby.jruby_puppet_schemas PoisonPill RetryPoisonPill))
```

Comment:
Interesting - how come the `:import` with a clojure namespace?  Can't we just reference the pills with `jruby-schema/PoisonPill`?

Not a big deal, just curious since I don't think I've seen this style before.